src/parsers/protobuf/main.ts

```diff
@@ -95,15 +94,17 @@ export default function parseProtobuf(
   const treeIndexNodeIdMap: TreeIndexNodeIdMap = [];
   const colorMap: ColorMap = [];
 
-  const handleWebNode = (webNode: any) => {
-    const { boundingBox, path, id } = webNode;
+  let count = 0;
```

Comment:
Should this maybe be inside the function to make the behavior deterministic also when we have multiple viewers/parses in the same project? Just makes it a bit easier to debug down the line.

Reply:
It _is_ inside the `parseProtobuf` function, so multiple viewers / parsers will have different counts.

Reply:
My bad :)